docs/examples/pytorch/resnet50/main.py

```diff
@@ -84,14 +84,16 @@ def __init__(self, batch_size, num_threads, device_id, data_dir, crop, dali_cpu=
         #let user decide which pipeline works him bets for RN version he runs
         if dali_cpu:
             dali_device = "cpu"
-            self.decode = ops.HostDecoder(device=dali_device, output_type=types.RGB)
+            # self.decode = ops.HostDecoder(device=dali_device, output_type=types.RGB)
```

Comment:
Remove

Reply:
I think TensorFlow RN50 example could be updated as well.

Reply:
Will do